src/zzz_od/operation/key_sim_runner.py

```
@@ -1,28 +1,53 @@
 from one_dragon.base.operation.operation_node import operation_node
 from one_dragon.base.operation.operation_round_result import OperationRoundResult
+from one_dragon.utils import os_utils
 from one_dragon.utils.i18_utils import gt
 from zzz_od.context.zzz_context import ZContext
 from zzz_od.operation.zzz_operation import ZOperation
 
 
+def find_key_sim_yml(work_dir: Path, plugin_dirs: list[Path], config_name: str) -> Path | None:
+    """定位键鼠脚本 yml 的读取路径。
+
+    优先级：用户 config/key_sim/<name>.yml > 插件目录 key_sim/<name>.yml。
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)